src/testing_system/internal/usecase/Eval.py

```
@@ -1,66 +1,410 @@
+    output: List[MetricValue] 
 
-    def execute(self, q: Question, a: Answer) -> List[MetricValue]:
-        metrics = []
+    check all the entities in the
+    testing_system/internal/domain/entities.py
+    """
+    
+    def __init__(self, 
+                 metrics: List[str], 
+                 bert_model: str = "bert-base-multilingual-cased",
+                 hallucination_model: str = None,
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)